Makefile

```diff
@@ -15,7 +15,7 @@ bf_base: bf_base.o
 	${CC} ${LDFLAGS} -o $@ $<
 
 bf_simple_yk.o: bf_simple_yk.c
-	${CC} ${CFLAGS} ${YK_CFLAGS} -c -o $@ $<
+	${CC} -O2 ${CFLAGS} ${YK_CFLAGS} -c -o $@ $<
```

Comment:
Hard-coding like this is a very bad plan.

Please set this in `CFLAGS` but also make CFLAGS overridable with `?=` as per Makefile conventions so that someone can override it with `-O0` etc.

Reply:
[b083eef](https://github.com/ykjit/ykcbf/pull/24/commits/b083eef2c6c96147253190db8e2059a820dbd832)